scripts/perfmon

```diff
@@ -1067,7 +1067,7 @@ def main():
             vm_uuid_list = rrd_updates.get_uuid_list_by_objtype('vm')
 
             # Remove any monitors for VMs no longer listed in rrd_updates page
-            for uuid in vm_mon_lookup:
+            for uuid in list(vm_mon_lookup.keys()):
```

Comment:
Please add a comment for readers to make it clear why list(dict_var.keys()) is needed here:
```suggestion
            # we use .pop() inside the loop, use list(dict_var.keys()):
            for uuid in list(vm_mon_lookup.keys()):
```